### DISCUSSION TITLE
Trailing ::: at bottom of rendered html pages

### DISCUSSION THREAD

**Discussion by jhelvy** [quarto-dev/quarto-cli#8931] [Q&A] [answered]
Hi everyone. On every page in Quarto websites I make, I have a trailing `:::` appearing at the bottom of every page. You can see an example on one of my class websites [here ](https://p4a.seas.gwu.edu/2024-Spring/). Every page has the `:::` at the bottom. Any idea what's causing that? Here's a screenshot in case it doesn't appear on your browser.

![Screen Shot 2024-02-28 at 8 35 32 PM](https://github.com/quarto-dev/quarto-web/assets/9012994/ef5ff0b9-226f-4e0e-a979-e9edf2e24c0a)

**Comment by jhelvy** [comment]
I should note that I also posted this question on the Posit community forums [here](https://community.rstudio.com/t/quarto-trailing-at-bottom-of-rendered-html-pages/182141), haven't seen a response yet.

**Comment by cscheid** [comment]
That's a stray div enclosure (from writing divs like `::: {} \n (content)\n ::: \n`, though it's hard to say what's causing it without a smaller repro.

**Comment by jhelvy** [comment]
Right, but it's on every page of my site, and on any site I make. Most of those pages don't have any div enclosures with `:::`, so it's hard to diagnose where it's coming from.

**Reply by cscheid** [reply]
It might be a bug on our side with incorrectly escaping (for example) something on the navbar or sidebar.

**Comment by jhelvy** [comment]
Strangely enough, it seems to only appear when deployed. I just built the site locally with `quarto render` and I don't see it anywhere. 

My workflow is to render the site using github actions and then deploy the rendered pages from github pages, so maybe somewhere in the rendering phase with github actions there is an issue?

**Reply by cscheid** [reply]
That is probably due to a difference in versions between your CI action and your local version of quarto. Looking at the HTML meta tags should give you the information.

**Comment by cderv** [comment]
Thanks for the report @jhelvy. I can reproduce with your repository and it seems to be an issue with `page-footer` processing. 

If you simplify your `page-footer` to just some text without HTML characters, it will disappear. So testing further, I can say it is linked to quotes. 

If you change external quaotes from `'` to `"` and internal ones from `"` to `'`, it also solves the issue. 

````diff
diff --git a/_quarto.yml b/_quarto.yml
index d69cf16..a74fa7d 100644
--- a/_quarto.yml
+++ b/_quarto.yml
@@ -83,7 +83,7 @@ website:
             href: https://github.com/emse-p4a-gwu/2024-Spring
   page-footer:
     center:
-      - text: '© 2024 John Paul Helveston <a href="LICENSE.html"><i class="fab fa-creative-commons"></i><i class="fab fa-creative-commons-by"></i><i class="fab fa-creative-commons-sa"></i></a><br><i class="far fa-calendar-alt"></i> {{< var weekday >}} | <i class="far fa-clock"></i> {{< var time >}} | <a href="{{< var room_url >}}"><i class="fa fa-map-marker-alt"></i> {{< var room >}}</a> | <a href="https://www.jhelvy.com"><i class="fas fa-user"></i> Dr. John Paul Helveston</a> | <a href="mailto:{{< var email >}}"><i class="fas fa-envelope"></i> {{< var email >}}</a> | <a href="{{< var repo >}}"><i class="fab fa-github"></i></a>'
+      - text: "© 2024 John Paul Helveston <a href='LICENSE.html'><i class='fab fa-creative-commons'></i><i class='fab fa-creative-commons-by'></i><i class='fab fa-creative-commons-sa'></i></a><br><i class='far fa-calendar-alt'></i> {{< var weekday >}} | <i class='far fa-clock'></i> {{< var time >}} | <a href='{{< var room_url >}}'><i class='fa fa-map-marker-alt'></i> {{< var room >}}</a> | <a href='https://www.jhelvy.com'><i class='fas fa-user'></i> Dr. John Paul Helveston</a> | <a href='mailto:{{< var email >}}'><i class='fas fa-envelope'></i> {{< var email >}}</a> | <a href='{{< var repo >}}'><i class='fab fa-github'></i></a>"

 format:
   HTML:
````

I can reproduce the issue with a `quarto create website` project, so I'll open an issue to track. 

Thanks again ! Github repo with DESCRIPTION are easily reproducible using git clone and **renv** - it really helps to debug ! Thank you.

**Reply by cderv** [reply, marked as answer]
I moved to `quarto-cli` repo and created an issue 
* https://github.com/quarto-dev/quarto-cli/issues/8932

closing this one now. You can follow the resolution there.
